src/dnx.common/utils.cpp

```diff
@@ -128,7 +128,7 @@ namespace dnx
         }
 
 #if defined (_WIN32)
-        const wchar_t* get_windows_version()
+        xstring_t get_windows_version()
```

Comment:
make it `std::wstring` - the convention I followed was to use native types in code that is platform specific and x-types in code that is platform agnostic.